sql/hive/pom.xml

```diff
@@ -72,6 +72,11 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk18on</artifactId>
+      <version>${bouncycastle.version}</version>
```

Comment:
Isn't it necessary to add `<scope>test</scope>`?

Reply:
I've checked, and since the default scope of this dependency is test, it's not necessary to add `<scope>test</scope>` here.

Reply:
Yes. Thanks!